clients/admin-ui/src/features/config-wizard/SuccessPage.tsx

```diff
@@ -49,7 +50,21 @@ const SuccessPage = ({
 
   const onFinish = () => {
     dispatch(setActiveSystem(undefined));
-    return features.plus ? router.push("/datamap") : router.push("/system");
+    const basePath = router.basePath || "/";
```

Comment:
Because this is the admin-ui, the basePath will always be `/`. that means you can simplify this bit of code to:

```typescript
const datamapRoute = resolveLink({
  href: "/datamap",
  basePath: "/"
});

features.plus ? router.push(datamapRoute.href) : router.push('/system')
```

I could also see an argument to do it the way you have now, since then these components would work regardless of if they are in the admin UI or in the datamap UI, but in that case, we should refactor this into a function so that all of the places that need it can use it without copy and pasting code